operators/src/processing/raster_vector_join/mod.rs

```diff
@@ -28,19 +29,37 @@ const MAX_NUMBER_OF_RASTER_INPUTS: usize = 8;
 
 /// The parameter spec for `RasterVectorJoin`
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct RasterVectorJoinParams {
     /// Each name reflects the output column of the join result.
     /// For each raster input, one name must be defined.
     pub names: Vec<String>,
 
-    /// Specifies which method is used for aggregating values
-    pub aggregation: AggregationMethod,
+    /// Specifies which method is used for aggregating values for a feature
```

Comment:
for each feature?

So this is the aggregation which is used to generate a single value for each feature? E.g. all pixels inside of a polygon?